config/sidekiq_scheduler.yml

```diff
@@ -215,6 +215,10 @@ Veteran::VSOReloader:
   class: Veteran::VSOReloader
   description: Pulls down current list of VSOs and updates accordingly 
 
+VBADocuments::DeploymentsNotifier:
+  at: "<%=0.seconds.from_now%>"
```

Comment:
What is the reason for 0.seconds.from_now?

Reply:
Runs on startup.